contrib/replicate-build/template/cog.yaml

```diff
@@ -0,0 +1,11 @@
+build:
+  gpu: true
+  python_version: '3.10'
+  python_packages:
+  - 'git+https://github.com/eth-sri/lmql@main#egg=lmql'
```

Comment:
Each version cog uploads is assigned a hash, but that's versioning of the outputs; if there's good support for versioning of the inputs, I'm unaware of it.

If the latest pypi-published version of lmql worked, I'd be tempted to use that instead of installing from source; however, right now latest is 0.0.6.x, which is not great for llama-2 models.